src/gtk/toga_gtk/app.py

```diff
@@ -1,85 +1,16 @@
 import asyncio
-import os
 import signal
 import sys
 
 try:
     import gi
 except ImportError:
-    # app.py is the first module that will be imported when you import toga_gtk.
-    #
-    # If Gtk can't be imported, it may be because we're in a virtualenv,
-    # and the system python libraries aren't visible. This can be fixed by
-    # creating a symlink into the site-packages
-    # Try creating a symlink to the system library location.
-    # base_packages_dir is where the packages installed by the package manager
-    # can be found.
-    # gi_system_install_path is where gi can be found in the packages dir.
-    # installer_command is the command the user can run to install gi.
     py_version = "%d.%d" % (sys.version_info.major, sys.version_info.minor)
-
     if sys.version_info.major == 3:
-        if os.path.isdir('/usr/lib64/python%s/site-packages/' % (py_version,)):
-            # Fedora
-            base_packages_dir = '/usr/lib64/python%s/site-packages/' % (py_version,)
-            gi_system_install_path = '/usr/lib64/python%s/site-packages/gi' % (py_version,)
-            installer_command = 'dnf install pygobject3 python3-gobject'
-        elif os.path.isdir('/usr/lib/python3/dist-packages/'):
-            # Ubuntu, Debian
-            base_packages_dir = '/usr/lib/python3/dist-packages/'
-            gi_system_install_path = '/usr/local/lib/python3/dist-packages/gi'
-            installer_command = 'apt-get install python3-gi'
-        elif os.path.isdir('/usr/lib/python%s/site-packages/' % (py_version,)):
-            # Arch
-            base_packages_dir = '/usr/lib/python%s/site-packages/' % (py_version,)
-            gi_system_install_path = '/usr/lib/python%s/site-packages/gi' % (py_version,)
-            installer_command = 'pacman -S python-gobject'
-        else:
-            raise RuntimeError("Unable to locate your Python packages dir.")
+        raise RuntimeError("Are you sure you have installed the Python GTK+ bindings? If not, run pip install pygobject")
```

Comment:
If we set the install requirements, the `pip install` part shouldn't be a problem - the problem will be if the underlying binary libraries aren't there... and I'm not sure if that's even *possible*  for GTK. It might be for Webkit and Cairo, though.

Reply:
Ok fixed.